hack/release.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Tools:
+# * ko: https://ko.build/
+# * helm: https://helm.sh/
+# * gh: https://cli.github.com/
+# * release-notes: https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md
+#
+# Externally provided variables
+# export OCI_REPOSITORY="quay.io/username"
+# export RELEASE_REPOSITORY="username/kubearchive"
+# export GITHUB_TOKEN="token-string"  # auth for 'release-notes'
+# export GH_TOKEN="token-string"  # auth for 'gh'
+#
+set -o errexit  # -e
+set -o xtrace  # -x
+
+# Variables
+export GIT_COMMITTER_NAME="github-actions[bot]@users.noreply.github.com"
+export GIT_COMMITTER_EMAIL="github-actions[bot]"
+export GIT_AUTHOR_NAME=${GIT_COMMITTER_NAME}
+export GIT_AUTHOR_EMAIL=${GIT_COMMITTER_EMAIL}
+export KO_DOCKER_REPO="${OCI_REPOSITORY}"
+export CURR_VERSION=$(cat ./VERSION)
+
+# release-notes environment variables
+export BRANCH="main"
+export START_SHA=$(git rev-list -n1 ${CURR_VERSION})
+export END_REV=${BRANCH}
+
+release-notes generate \
+    --required-author="" \
+    --format json \
+    --output ./release-notes.json \
+    --repo kubearchive --org kubearchive
+go run hack/get-next-version.go \
```

Comment:
Since these are multi-line commands, I suggest adding a blank line between these two commands to avoid confusion (at first I thought it was one long command):

```suggestion
    --repo kubearchive --org kubearchive

go run hack/get-next-version.go \
```